components/Modal.css

```diff
@@ -0,0 +1,86 @@
+/* color variables */
+
+:root {
```

Comment:
Instead of defining root color at different files you can have it inside one main css file and import it in rest of the CSS files.

Reply:
that's great idea